src/scss/components/_spinner.scss

```diff
@@ -0,0 +1,17 @@
+.lg-spinner {
+  margin-left: auto;
```

Comment:
I think the spinner class itself shouldn't care about where it sits on the screen. While our current implementation has it there, we don't know if all future versions will put it in the same place.